src/cljs/org/broadinstitute/firecloud_ui/common/entity_table.cljs

```diff
@@ -74,8 +74,14 @@
                                                (common/attribute-list? attr-value)
                                                (let [items (map render-list-item (common/attribute-values attr-value))]
                                                  (if (empty? items)
-                                                   "0 items"
-                                                   (str (count items) " items: " (join ", " items))))
+                                                    (cond (= (str selected-entity-type) ":sample_set") "0 samples"
+                                                          (= (str selected-entity-type) ":pair_set") "0 pairs"
+                                                          (= (str selected-entity-type) ":participant_set") "0 participants"
+                                                          :else  "0 items")
+                                                    (cond (= (str selected-entity-type) ":sample_set") (str (count items) " samples")
```

Comment:
After this change, we don't list the samples here.  Is that what we want?

PO decision, I can see arguments for either way.  I just didn't see this change in the story.